Ez namey.js

```diff
@@ -18,5 +18,7 @@ let secondWords = ["Solutions",
 "Networks",
 "Dynamics",
 "Strategies"];
-let startupName = firstWords[4] + " " + secondWords[8]
-console. log(`The startup: "${startupName} " contains ${startupName.length} characters`.);
\ No newline at end of file
+const randomNumber1 = Math.floor(Math.random() * 10) +1;
+const randomNumber2 = Math.floor(Math.random() * 10) +1;
```

Comment:
You have to remove the `+1` at the end, as because of it you would never get index 0 in the arrays ("Innovative" and "Solutions"), and you would sometimes be getting index 10, but there is no such element so it will be `undefined`. Remember that in JavaScript arrays start counting from index 0.
[Try this CodePen](https://codepen.io/menian/pen/BaeZQaY) and see when the index happens to be 0 or 9  that it would correspond to the first and last item in the array. Then change it to `+1` and see the results. In bottom left corner there is a button to open the "console" where you can see the logs.